packages/material-ui/src/Drawer/Drawer.test.js

```diff
@@ -299,4 +299,20 @@ describe('<Drawer />', () => {
       expect(getAnchor(theme, 'right')).to.equal('left');
     });
   });
+
+  describe('zIndex', () => {
+    it('should set correct zIndex on the root element', () => {
+      const theme = createTheme();
+      mount(
+        <ThemeProvider theme={theme}>
+          <Drawer open>
+            <div />
+          </Drawer>
+        </ThemeProvider>,
+      );
+      expect(screen.getByRole('presentation')).toHaveComputedStyle({
+        zIndex: String(theme.zIndex.drawer),
+      });
+    });
+  });
```

Comment:
Sorry, I merged too quickly. This is not great. I have fixed in e75b440399146899a12cd5377774923d1a24fc9e.